tasks/check-data-keys/README.md

```diff
@@ -3,10 +3,10 @@
 Tekton task to check that all required information is present in the data file in order to make a
 call to the specified system(s). If any required keys are missing, the task will fail.
 
-For example, if `advisory` is passed as a system and the data file does not have all the required
-advisory keys, the task will fail.
+For example, if `releaseNotes` is passed as a system and the data file does not have all the required
```

Comment:
I understand that you're just replacing "advisory" with "releaseNotes". But I am confused about the word "system" here. The first paragraph says "in order to make a
call to the specified system(s)". And the last paragraph says that advisory/releaseNotes is the system. How do you make a call to releaseNotes? What does it mean? So maybe just the first paragraph needs some rewording? Any idea what it should say?

Reply:
If you have a better word for [releaseNotes, fbc, images, mapping, maybe I am missing something else?], I am happy to change it. System was just the word I came up with for it

Reply:
I'm fine with system. But with this meaning, I am not sure about "make a call to the specified system(s)" (I guess I should have commented there and not here, but at first I thought this second occurrence of "system" is maybe the one to change).

So maybe replace "make a call to" with "use" or "process"? Like: "Tekton task to check that all required information is present in the data file in order to use the specified system(s)."

Reply:
I would have called it `feature` maybe and remove the `make a call` comment.

Something like:
```
Tekton task to check that all required information is present in the data file in order to make use of the specified features. If any required keys are missing, the task will fail.
```

Reply:
I'm not sold on using the word feature, I think it is a very commonly used word and is actually overused. I do like Martin's suggestion though (that it seems David agrees with) and pushed a change with it

Reply:
Sounds good to me, thanks.